coalib/coala.py

```diff
@@ -61,13 +66,19 @@ def main():
 
             return 0
```

Comment:
I'd move the show ... stuff up so first you check for those. Thing is, they don't need the `run_coala` import and that import takes like 0.2 seconds which in that case is like half of the execution time. So move the show... stuff up, convert all the `elif` into `if` because you return anyway in each condition and do the import of the modes *after* the show stuff